src/components/inputs/Slider/Slider.tsx

```diff
@@ -111,10 +118,19 @@ export function Slider({
     dispatch({ type: 'SET_TRACK_RECT', payload: { left, width } });
   }, [dragging]);
 
+  useEffect(() => {
+    if (!isFirstRender.current) {
+      dispatchChangeEvent(onChange);
+    } else {
+      isFirstRender.current = false;
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Nit: could you leave a comment on why this is being ignored?

Reply:
Sure, added